Cython/Compiler/Symtab.py

```diff
@@ -31,6 +29,165 @@
     '_Bool', '_Complex'', _Imaginary', 'inline', 'restrict',
 }
 
+# Builtins python 3.12.2
+
+builtins_attrs = set(['ArithmeticError',
```

Comment:
Please place this list as follows, to make it easy to compare, extend and comment in/out:
```python
KNOWN_PYTHON_BUILTINS = frozenset([
    'first',
    …
    'last',
])
```